package.json

```diff
@@ -1,5 +1,28 @@
 {
+  "name": "@keep-network/coverage-pools",
+  "version": "0.0.1",
+  "files": [
+    "contracts/",
+    "!contracts/test/",
+    "deploy/",
+    "deployments/",
+    "export.json"
+  ],
+  "scripts": {
+    "build": "hardhat compile",
+    "deploy": "hardhat deploy --export export.json",
+    "lint": "npm run lint:js && npm run lint:sol",
+    "lint:fix:js": "eslint . --fix",
+    "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
+    "lint:js": "eslint . ",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
+    "test": "hardhat test",
+    "test:system": "NODE_ENV=system-test hardhat test ./test/system/*.test.js",
+    "test:unit:coverage": "hardhat coverage"
+  },
   "dependencies": {
+    "@keep-network/keep-core": "^1.8.0-dev",
+    "@keep-network/tbtc": "^1.1.2-dev",
```

Comment:
For local development- since we rely on external packages we should link them in the `install.sh` script eg: 
```
yarn link @keep-network/tbtc
yarn link @keep-network/keep-core
```


Note that `tbtc` and `keep-core` use `npm` instead of `yarn` so we have to create links via `yarn link` for `tbtc` and `keep-core`. But I think it should be done in [`keep-core/scripts/start_dashboard.sh`](https://github.com/keep-network/keep-core/blob/main/scripts/start_dashboard.sh)

Reply:
Great catch! 

I added linking in the `install.sh` script. It assumes the links are made before running the script. So we will have to create them in `local-setup` similarly we do for other repos ([example](https://github.com/keep-network/local-setup/blob/5fd3c299943a89d4281ec2021ce863c21015faa6/install-tbtc.sh#L46-L50)).

And for the dashboard exactly as you suggested in `keep-core/scripts/start_dashboard.sh`.